### PR TITLE
Add additional maintainers to avoid single point of failure

### PR DESCRIPTION
**Pull request by MarkWieczorek**
This feedstock has only a single maintainer. To avoid the case where this maintainer is unavailable to attend to duties related to this repo because of other engagements, I suggest adding two additional maintainers. The first is @mreineck, who is the lead developer of ducc0. The second is myself (@markwieczorek), who makes use of this project for the pyshtools project. @ickc, if you have any questions, please feel free to ask.

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mreineck** [comment]
While I'd certainly be happy to improve redundancy, I'm not sure that I could be a good maintainer for this package. The whole conda machinery is a complete mistery to me, and I most likely could not make informed decisions about most of the PRs.

**Comment by MarkWieczorek** [comment]
I am not a conda expert either, but we could always find someone better later! Do you want to try to ask around and see if you can find anyone better?

**Comment by mreineck** [comment]
Well, you knew the right incantations to enable the M1 builds, which I already consider impressive! :-)

As long as it is understood that I cannot do much more than perhaps accept PRs for new `ducc0` releases, I'm fine with the PR as it is.

**Comment by MarkWieczorek** [comment]
I spoke with @ickc over email, and it turns out that he is no longer a member of conda-forge and can't merge PRs here.

@conda-forge/core : would it be possible to accept this PR that adds additional maintainers to this package?

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
I can merge but....

I would like to ask that one of you two address one of the outstanding PRs in adding yourself as a maintainer. It seems to be behind a release or two and that would be a good place to start maybe?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @ickc
\ No newline at end of file
+* @MarkWieczorek @ickc @mreineck
```
Comment (hmaarrfk) [inline .github/CODEOWNERS:1]: This was the change i needed.

**Comment by MarkWieczorek** [comment]
Thanks for the help. I think that the first thing we need to do is accept the osx-arm64-migration PR (https://github.com/conda-forge/ducc0-feedstock/pull/29). Its my hope that all the failing checks will then pass.

And if you know of anyone that wants to to be a maintainer (such as yourself!), please feel free to add their names. I only know the basics of conda-forge packaging, and I stepped in only because there wasn't an active maintainer here...

**Comment by hmaarrfk** [comment]
You should both have access now

**Pull request closed without merging**